CHANGELOG.rst

```diff
@@ -1,6 +1,15 @@
 Changelog for Mantarray Desktop App
 ===================================
 
+1.1.3 (unreleased)
+------------------
```

Comment:
there are two `1.1.3 unreleased`

Reply:
This should be 1.1.4. That's the version in the `packag.json`